README.md

```diff
@@ -3,3 +3,19 @@
 This repository is cataloging University Library access to scholarly literature.
 Scholarly articles are identified using their DOIs.
 The impetus for this project was [this discussion](https://github.com/greenelab/scihub-manuscript/issues/21 "Potential followup: comparison to authorized access") on the Sci-Hub Coverage Study.
+
+# License
+
+The data files in this repository are released under the CC0 1.0 public domain dedication ([`LICENSE-CC0.md`](LICENSE-CC0.md)). These include files matched by the following glob patterns:
+
+- `*.tsv`
+- `*.sqlite`
+- `*.xml`
+
+The source code in this repository is released under a BSD 3-Claue license ([`LICENSE-BSD-3-CLAUSE.md`](LICENSE-BSD-3-CLAUSE.md)). These include files matched by the following glob patterns:
```

Comment:
Right now, we can accidentally get to a state where there are unlicensed files. I think it makes most sense to start with the BSD globs. Then say everything not matched above, which will mostly be datasets, are released under CC0.